src/bindings/python/wheel/requirements-dev.txt

```diff
@@ -1,3 +1,4 @@
+-r ../src/compatibility/openvino/requirements-dev.txt
```

Comment:
we can build wheel without old API.

Reply:
That would be better! cmake currently fails on ov_check_python_build_conditions if you do not have cython installed  (with -DENABLE_PYTHON=ON -DENABLE_WHEEL=ON). It would be great if this would just show a message that only OpenVINO Runtime API 2.0 will be enabled.

Reply:
Our cmake treats explicit `ENABLE_PYTHON` as "build both 1.0 and 2.0 APIs"
If you don't specify `ENABLE_PYTHON`, but requirements for 2.0 API  are satisfied, then only 2.0 API will be built.